Quest Behaviors/ForcedMount.cs

```diff
@@ -110,12 +110,19 @@ public ForcedMount(Dictionary<string, string> args)
 
             else
             {
-                Mount.FlyingMounts.First().CreatureSpell.Cast();
+                if (!string.IsNullOrEmpty(Styx.Helpers.CharacterSettings.Instance.FlyingMountName))
+                {
+                    foreach (var mount in Mount.FlyingMounts.Where(mount => mount.Name == Styx.Helpers.CharacterSettings.Instance.FlyingMountName))
+                        Mount.FlyingMounts.FirstOrDefault(m => m.CreatureSpellId == mount.CreatureSpellId)?.CreatureSpell.Cast();
+                }
+                else
+                    Mount.FlyingMounts.First().CreatureSpell.Cast();

```

Comment:
All of this can use [Flightor.MountHelper.MountUp](http://docs.honorbuddy.com/html/9a7d751f-71d6-a362-aba6-d6ee192432be.htm) instead, I believe.